ndt7/results/metadata.go

```diff
@@ -14,23 +12,13 @@ type metadata map[string]string
 // serverKeyRe is a regexp that matches any server related key.
 var serverKeyRe = regexp.MustCompile("^server_")
 
-// initMetadata initializes |*meta| from |localEpnt|, |remoteEpnt|, the |values|
-// provided using the query string, and the |subtest| name. Returns an error
-// on failure.
-func initMetadata(m *metadata, localEpnt, remoteEpnt, uuid string, values url.Values, subtest string) {
+// initMetadata initializes |*meta| from |values| provided from the original
+// request query string.
+func initMetadata(m *metadata, values url.Values) {
 	for k, v := range values {
 		if matches := serverKeyRe.MatchString(k); matches {
 			continue // We MUST skip variables reserved to the server
 		}
-		if len(v) != 1 {
-			continue // We SHOULD ignore duplicate keys
-		}
 		(*m)[k] = v[0]
 	}
-	(*m)["server_name"] = "ndt-server"
```

Comment:
In a distant future we may want to have another server implementation. Does it make sense to modify the top-level data structure to also include the server name?

Reply:
What would a top-level field for "ServerName" provide that the git commit and version fields do not?